test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -274,38 +273,97 @@ public void Should_Write_History_For_Audited_Entities_Update()
         }
 
         [Fact]
-        public void Should_Write_History_For_Audited_Entities_Update_Owned()
+        public void Should_Write_History_For_Owned_Entity_Of_Audited_Entities_Create()
         {
-            /* Blog has Audited attribute. */
+            // Blog is the owner of BlogEx and has Audited attribute.
+            // Therefore, BlogEx should follow Blog and have entity history.
 
-            int blog1Id;
-            var newValue = "blogger-2";
-            string originalValue;
+            WithUnitOfWork(() =>
+            {
+                // since owned entities is not available via DbContext -> DbSet,
```

Comment:
The point here is trying to explain that we can’t have DbSet<BlogEx> in DbContext, therefore we need to load it via the owner DbSet.

How about `Since owner entities are not available via DbContext`?

Reply:
The point here is trying to explain that we can’t have DbSet<BlogEx> in DbContext, therefore we need to load it via the owner DbSet.

How about `Since owner entities are not available via DbContext`?

Reply:
**Owner** entities are. Do you mean **owned** entities don't have `DbSet` instead?

Reply:
Yes, i meant Owned Entities do not have DbSet